Root/OptimizationDump.cxx

```diff
@@ -377,13 +377,17 @@ EL::StatusCode OptimizationDump :: execute ()
   }
 
   if(!m_inputMET.empty() && !m_inputJets.empty()){
-    m_effectiveMass = VD::Meff_inclusive(in_met, in_jets, in_muons, in_electrons)/1000.;
+    
+    ConstDataVector<xAOD::JetContainer> presel_jets = VD::subset_using_decor(in_jets, VD::decor_vd_pass_Presel, 1);
+    //std::cout << in_jets->size() << " " << presel_jets.size() << std::endl;
+    m_effectiveMass = VD::Meff_inclusive(in_met, presel_jets.asDataVector(), in_muons, in_electrons)/1000.;
```

Comment:
Oh good point! I was thinking of how 1L (Loic or LG?) had pT > 40 GeV -- so it would just be that only same(similar) meff is calculated for the subset of events with pT > 30 GeV... So perhaps presel is the way to go here.

This raises the question -- if we add a preselection on Meff -- what goes into that calculation in `Preselect.cxx`?